src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/resources/standard.properties

```diff
@@ -270,6 +274,18 @@ doclet.navClassUse=Use
 doclet.Error_in_grouplist=Bad -group option: {0} {1}
 doclet.Groupname_already_used=In -group option, group name already used: {0}
 doclet.Same_element_name_used=Element name or pattern used twice: {0}
+doclet.PreviewLeadingNote=<code>{0}</code> relies on preview features of the Java platform:
+doclet.Declared_Using_Preview=<code>{0}</code> is declared using <em>{1}</em>, a preview feature of the Java language ({2}).
+doclet.PreviewAPI=<code>{0}</code> refers to one or more preview APIs:  {1}.
+doclet.ReflectivePreviewAPI=<code>{0}</code> refers to one or more reflective preview APIs:  {1}.
+doclet.UsesDeclaredUsingPreview=<code>{0}</code> refers to one or more types which are declared using a preview feature of the Java language: {1}.
+doclet.PreviewTrailingNote1=Programs can only use <code>{0}</code> when preview features are enabled.
+doclet.PreviewTrailingNote2=Preview features may be removed in a future release, or upgraded to permanent features of the Java platform.
+doclet.Declared_Using_Preview.RECORD=Records
+doclet.Declared_Using_Preview.SEALED=Sealed Classes
+doclet.Declared_Using_Preview.SEALED_PERMITS=Sealed Classes
+doclet.PreviewPlatformLeadingNote=<code>{0}</code> is a preview API of the Java platform.
+doclet.ReflectivePreviewPlatformLeadingNote=<code>{0}</code> is a reflective preview API of the Java platform.
```

Comment:
HTML in resource files is bad.  It would be marginally better to move the HTML to the value being substituted (using strings from Content nodes) and even better to use a method that substitutes Content nodes into a resource string (Not sure if that method exists, but it could/should).